### PR TITLE
ELEMENTS-992: run tests with saucelabs

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -66,20 +67,21 @@ pipeline {
...
     stage('Run tests') {
       steps {
+        setGitHubBuildStatus('test', 'Unit tests', 'PENDING')
         container('nodejs') {
           echo """
           ---------
           Run tests
           ---------"""
-          sh 'npm run test -- --debugBrowsers=ChromeHeadlessNoSandbox'
+          sh 'SAUCE_USERNAME=nuxeo-elements SAUCE_ACCESS_KEY="$(jx step credential -s saucelabs-elements -k key)" npm run test'
```
Comment (troger) [inline Jenkinsfile]: Wouldn't it be clearer to use `withEnv` and/or `withCredentials`?

File: Jenkinsfile [hunk 2]
```diff
@@ -66,20 +67,21 @@ pipeline {
...
         }
       }
       post {
         success {
-          setGitHubBuildStatus('webpack', 'Webpack build', 'SUCCESS')
+          setGitHubBuildStatus('test', 'Unit tests', 'SUCCESS')
         }
         failure {
-          setGitHubBuildStatus('webpack', 'Webpack build', 'FAILURE')
+          ssetGitHubBuildStatus('test', 'Unit tests', 'FAILURE')
```
Comment (nmpcunha) [inline Jenkinsfile]: Typo here `set`instead of `sset`?

File: package.json [hunk 3]
```diff
@@ -46,7 +46,7 @@
     "karma-firefox-launcher": "^1.1.0",
     "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.5",
-    "karma-sauce-launcher": "^2.0.2",
+    "karma-sauce-launcher": "^1.2.0",
```
Comment (nmpcunha) [inline package.json]: Why are we downgrading the version?

File: Jenkinsfile (hunk 1 above)
Comment (nelsonsilva) [reply Jenkinsfile]: done 👍

**Pull request merged**